apps/settings/js/app_storage.js

```diff
@@ -28,11 +28,13 @@ var AppStorage = (function AppStorage() {
   function attachListeners() {
     _appStorage.addEventListener('change', getSpaceInfo);
     window.addEventListener('localized', getSpaceInfo);
+    window.addEventListener('panelready', getSpaceInfo);
```

Comment:
We should only update the info when users are navigating to the application storage panel. This can be done by checking `e.detail.current` of the panel ready event.